modules/file-system/parallelstore/scripts/mount-daos.sh

```diff
@@ -65,6 +65,9 @@ chmod 777 "$local_mount"
 fuse_config=/etc/fuse.conf
 sed -i "s/#.*user_allow_other/user_allow_other/g" $fuse_config
 
+# make sure limit of open files is high enough for dfuse (1M of open files)
+ulimit -n 1048576
```

Comment:
Is this for ubuntu only ?

Reply:
I guess this is for Ubuntu, RHEL9 and probably Debian. @jolivier23 suggests, that only Rocky Linux (or HPC image) has a sane default.